content/automate/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/check_service_retired.rb

```diff
@@ -16,6 +16,13 @@ def main
 
               private
 
+              def log_task_info(task)
+                @handle.log('info', "Service Retire Task <#{task.id}> <#{task.description}> is not retired, setting retry.")
+                task.miq_request_tasks.each do |t|
+                  @handle.log('info', "  Service Retire Task <#{task.id}> waiting on <#{t.request_type}> Task <#{t.id}> <#{t.description}>") if t.state != 'finished'
```

Comment:
Service Retire Task <#{task.id}> `is` waiting on ...

Reply:
Good catch, will change this.  Thanks